pkg/operator/config.go

```diff
@@ -45,9 +45,13 @@ type Config struct {
 }
 
 type ReloaderConfig struct {
-	CPU    string
-	Memory string
-	Image  string
+	CPU           string
```

Comment:
I would get rid of the `CPU` field (same for `Memory`) and move the logic of choosing between the global value and request/limit values in main().

Reply:
I'll make a temporary variable to replace the now removed `cfg.ConfigReloader.CPU` and `cfg.ConfigRelaoder.Memory`.

Reply:
A number of tests break with the removal of ReloaderConfig.CPU and ReloaderConfig.Memory. Correcting...

Reply:
Fixed.